### ISSUE TITLE
Feature Request - Divide wall more than 4 times, especially portals

### ISSUE THREAD

**Issue by illyaine** [MontyTRC89/Tomb-Editor#623]
Currently you can only subdivide a wall 4 times and a wall with a portal only once.
It would be better if it could be subdivided more often, especially the wall near the portal.

![2F1F6E1D-F2EF-4177-91FF-B828A2746741](https://user-images.githubusercontent.com/915558/196019868-e3501c7f-9dd1-477d-b24c-95da8ac98b28.jpeg)

![CC98A2F6-5403-4EC2-B6D6-564F7AD5B6BB](https://user-images.githubusercontent.com/915558/196019883-975cd5b2-19fc-44dd-bbdc-00f8d8d22f06.jpeg)

**Comment by Lwmte** [comment]
Duplicate of #48
